helpdesk/overview/sla.rst

```diff
@@ -0,0 +1,45 @@
+==============================
+Service Level Agreements (SLA)
+==============================
+
+Service Level Agreements (SLA) are commitments you make with your customers to outline how a
+service is delivered. It bolsters trust between you and your customers as it makes clear what
+needs to be done, to what standard, and when.
+
+Create your policies
+====================
+
+| First, enable the feature on the settings of the team you would like policies to be applied,
+  going to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`.
```

Comment:
```suggestion
| First, go to :menuselection:`Helpdesk --> Configuration --> Helpdesk teams` and enable the feature in the appropriate team's settings.
```

Reply:
I see the value of your suggestion, but I will keep the way I wrote because it states that you choose at which team you would like the policies to be applied. And this is an important piece of info (:

Reply:
:ok_hand: